Use Case: Plant Database Update Notification (#223)
ElektraInitiative/PermaplanT: doc/usecases/plant_positions_update.md

# Use Case: Plant Database Update Notification

## Summary

- **Scope:** Notifications & Plant Management 
- **Level:** User Goal
- **Actors:** App User
- **Brief:** Notify users upon login about updates to the plant database when a plant's positioning has changed.
- **Status:** Draft
- **Assignee:** Ramzan

## Scenarios

- **Precondition:**
 - The user has opened the app successfully logged in.
 - The plant database has been updated since the user's last login.
- **Main success scenario:**
 - Upon login, the user receives a notification about the updated plant database.

>  - The notification highlights plants that have changed positioning.

Reviewer: There is some misunderstanding: it is not about positioning but various other attributes of the plants.

```suggestion
 - The notification highlights plants that have changed.
```

>  - The user can review the updated plant positioning and adjust their garden layout accordingly.

Reviewer: ```suggestion
 - The user adjusts their garden layout accordingly.
 - The user confirms that the updates were considered and goes back to normal mode.
```

- **Alternative scenario:**

>  - The user dismisses the notification without reviewing the updated plant positioning recommendations. The notification is accessible later from a designated area in the app.

Reviewer: ```suggestion
 - The user dismisses the notification without reviewing the updated plant positioning recommendations. The notification is accessible later from a designated area in the app.
 - The user decides to not receive updates of changed plants in the future.
```
